app_builder/README.md

```diff
@@ -0,0 +1 @@
+# AppBuilder
```

Comment:
I'd like to eventually extract it out but let's iterate on it as part of the Livebook repo for the time being.